docs/index.html

```diff
@@ -133,10 +133,10 @@ <h1 class="page-title">
         </h1>
         <table-component
             :data="[
-            { firstName: 'John', lastName: 'Lennon', instrument: 'Guitar', birthday: '04/10/1940', songs: 72 },
-            { firstName: 'Paul', lastName: 'McCartney', instrument: 'Bass', birthday: '18/06/1942', songs: 70 },
-            { firstName: 'George', lastName: 'Harrison', instrument: 'Guitar', birthday: '25/02/1943', songs: 22 },
-            { firstName: 'Ringo', lastName: 'Starr', instrument: 'Drums', birthday: '07/07/1940', songs: 2 },
+            { key: 1, firstName: 'John', lastName: 'Lennon', instrument: 'Guitar', birthday: '04/10/1940', songs: 72 },
```

Comment:
Sorry, I still don't understand what this does. When I run the example locally, I'm not getting any warnings.

I understand why `key` is necessary and when it gives warnings. Adding it to the `li` in pagination makes perfect sense, since Vue throws a warning when iterated elements don't have a key. However, this has nothing to do with the contents of an object. I don't understand why adding a `key` property to an object would solve anything, `key` is for elements, not data.